ssh/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM centos:7
+
+# Install OpenSSH
+RUN yum install -y epel-release.noarch
+RUN yum upgrade -y
+RUN yum -y install openssh-server openssh-clients sudo
+RUN yum clean all
+
+# Add user
+RUN useradd -s /bin/bash -g root -G wheel -u 1000 sshtest 
+RUN echo sshtest:Testing | chpasswd
+
+# Server config
+RUN /usr/bin/ssh-keygen -A
+RUN sed -i 's/#PasswordAuthentication no/PasswordAuthentication yes/Ig' /etc/ssh/sshd_config
+RUN sed -i 's/#PubkeyAuthentication no/PubkeyAuthentication yes/Ig' /etc/ssh/sshd_config
+RUN echo "StrictHostKeyChecking no" >> /etc/ssh/ssh_config
+
+# Make directory to host the public keys
+RUN mkdir ~/.ssh
+RUN chown -R sshtest:root ~/.ssh; chmod -R 700 ~/.ssh
```

Comment:
not sure if this will work as expected. This changes the .ssh folder in /root and not in the home of the sshtest user